runtime/doc/options.txt

```diff
@@ -1077,8 +1074,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 			{not available when compiled without the |+linebreak|
 			feature}
 	This option lets you choose which characters might cause a line
-	break if 'linebreak' is on.  Only works for ASCII and also for 8-bit
-	characters when 'encoding' is an 8-bit encoding.
+	break if 'linebreak' is on.  Only works for ASCII characters.
```

Comment:
utf8 is an 8-bit encoding so this should say

    Works for ASCII and 8-bit characters.

(or maybe just delete the "works for ASCII" sentence.)